cluster/evaluator/cc-evaluator.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: minigo-cc-evaluator-$MODEL_WHITE_NUM-$MODEL_BLACK_NUM
+spec:
+  parallelism: 10
+  completions: 10
+  template:
+    metadata:
+      name: gpu-evaluator
+    spec:
+      containers:
+      - name: evaluator
+        image: gcr.io/$PROJECT/minigo-cc-evaluator:$VERSION_TAG
```

Comment:
i didn't extract the 'minigo-cc-evaluator' out to an env var because i'm not sure why we parameterized the container names to environment variables in the first place